lib/ansible/module_utils/common/respawn.py

```diff
@@ -92,6 +92,6 @@ def _create_payload():
     runpy.run_module(module_fqn, init_globals=dict(_respawned=True), run_name='__main__', alter_sys=True)
     '''
 
-    respawn_code = respawn_code_template.format(module_fqn=module_fqn, modlib_path=modlib_path, smuggled_args=smuggled_args.strip())
+    respawn_code = respawn_code_template.format(module_fqn=module_fqn, modlib_path=modlib_path, smuggled_args=to_native(smuggled_args).strip())
```

Comment:
I assume you need `to_text` here and not `to_native` as that would result in a `b''` on Python 2.